network/src/commonMain/kotlin/com/wire/kalium/network/api/base/unbound/acme/ACMEApi.kt

```diff
@@ -144,9 +119,10 @@ class ACMEApiImpl internal constructor(
         const val DEX_PORT = "5556"
 
         const val PATH_DEX_CONFIGURATION = "dex/.well-known/openid-configuration"
-        const val PATH_ACME_DIRECTORIES = "acme/wire/directory"
+        const val PATH_ACME_DIRECTORIES = "acme/google-android/directory"
```

Comment:
There's a special path for android? Should it be different for other platforms? Then we need inject it or do an expect/actual.

Reply:
yes, since they have different oauth structure and keys.

Reply:
later in the real environment will have these fields on the server side.
Will add a todo for Kalium that we need to manage it for different platforms.